content/overview/pricing/pricing-terminology.md

```diff
@@ -23,7 +23,7 @@ A system maps to an [“org” in cloud.gov and Cloud Foundry](http://docs.cloud
 
 ### Support {#support}
 
-“Support” is support to use the platform as intended, on a best-effort basis. We have no existing service-level agreement. We plan to always publish metrics that give agencies the ability to make an informed choice about whether to use cloud.gov or an alternative PaaS solution based on our actual track record.
+“Support” is support to use the platform as intended, on a best-effort basis. We have no existing service-level agreement (SLA). We plan to always publish metrics that give agencies the ability to make an informed choice about whether to use cloud.gov or an alternative PaaS solution based on our actual track record.
```

Comment:
Oh boy, it looks like this whole page really needs reworking. It's not very helpful to explain complicated topics by giving a list of terms and defining them, instead of centering the problem the user is trying to solve and explaining what they need to know. I'll give it a shot as part of working on the pricing info. :/